mywrk.lua

```diff
@@ -0,0 +1,86 @@
+wrk.method = "GET"
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::Duration;
+
+use url::Url;
+
+use crate::blueprint::Blueprint;
+use crate::config::reader::ConfigReader;
+use crate::config::Config;
+use crate::http::ServerContext;
+
+pub struct ConfigLoader {
+  file_paths: HashMap<String, u64>,
```

Comment:
do we need a hashmap?

Reply:
Yes in case of any failure, we are increasing timeout for each path/url, so this map stores the path and it's timeout

Reply:
why should it be different?